src/main/java/de/tum/in/www1/artemis/web/rest/ExerciseResource.java

```diff
@@ -106,12 +107,14 @@ public class ExerciseResource {
 
     private final ExerciseHintService exerciseHintService;
 
+    private final ParticipationVCSAccessTokenRepository participationVCSAccessTokenRepository;
```

Comment:
I am not sure if am missing something. Why was this added if it's not being used anywhere

Reply:
I used it in a previous prototype, I removed it again :+1: